web/viewer.css

```diff
@@ -17,6 +17,10 @@
 
 :root {
   --dir-factor: 1;
+  /*#if !MOZCENTRAL*/
```

Comment:
I'm probably misunderstand something here, but if I look at https://developer.mozilla.org/en-US/docs/Web/CSS/float#browser_compatibility I see that Firefox is the only one that _does_ support `float-inline-*` values, so shouldn't these `if` statements (also the other ones below) be inverted to `if MOZCENTRAL` so that we keep them for Firefox?

Moreover, how does this affect non-Firefox targets? As in, doesn't this break the viewer if we remove them altogether without a replacement?